### ISSUE TITLE
Include a activeHighlightIndex prop to the component

### ISSUE THREAD

**Issue by blessenm** [bvaughn/react-highlight-words#13]
#I am trying to create something like the built in search of the chrome browser. You can notice that the first match will be orange while the rest of the matches are in yellow. We can now cycle through the matches by clicking next or previous button and this would update the background to orange for the active match.

I implemented this in my fork by doing something like this

```
let highlightCount = 0;
{chunks.map((chunk, index) => {
        const text = textToHighlight.substr(chunk.start, chunk.end - chunk.start)

        if (chunk.highlight) {
          highlightCount++;
          highlightClassName = classnames( highlightClassName , { active: highlightCount == activeHighlightIndex })
          return (
            <HighlightTag
              className={highlightClassName}
              key={index}
              style={highlightStyle}
            >
              {text}
            </HighlightTag>
          )
        } else {
          return (
            <span key={index}>{text}</span>
          )
        }
      })}
```

**Comment by bvaughn** [comment]
This is a little out of scope of my initial intentions for this component. Feel free to submit a PR if you'd like to introduce support for this and I'll be happy to review it. I don't have the bandwidth to implement it myself though. 😄

**Comment by blessenm** [comment]
Yes I will work on a PR sometime next week and submit it for review.

**Comment by bvaughn** [comment]
Thanks so much for your contribution 😄

This feature has been released as version 0.6.0.

One thing to note is that I renamed `activeHighlight*` to `active*`. The "highlight" part seemed redundant and made the property names a bit long. 😁

**Comment by blessenm** [comment]
Awesome. Thanks for the quick review.
